app/src/main/java/com/T05/krowdtrialz/ui/experimentDetails/ExperimentStatistics.java

```diff
@@ -74,16 +80,100 @@ public View onCreateView(LayoutInflater inflater, ViewGroup container,
      *  Ricky Au
      */
     private void populateTrialResults(View view) {
+        String type = experiment.getType();
+        String stdString = "NaN";

```

Comment:
Also maybe here do this instead so if we want something else we can change more easily:
```
Float f = Float.NaN;
String nan = f.toString();

String stdString = nan;
...
```